unit-testing/index.md

```diff
@@ -1,7 +1,20 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Testing in software development​
+    - testing role
+    - benefits
+    - testing scope
+2. Testing Pyramid
+3. Unit tests
+    - pros and cons
```

Comment:
what do you think about a common overview the most popular testing libs for RN?

Reply:
Agreed. I think it would be nice to highlight the Jest + RN testing library combo.